projects/demo/src/modules/app/utils/sort-pages.ts

```diff
@@ -0,0 +1,39 @@
+import {TuiDocPage, TuiDocPageGroup, tuiIsPageGroup} from '@taiga-ui/addon-doc';
+
+export function tuiSortPages<T extends TuiDocPage | TuiDocPageGroup>(
```

Comment:
also you can move it to `@taiga-ui/addon-doc` 
this may be useful for other developers